tests/phpunit/tests/query/lazyLoadTermMeta.php

```diff
@@ -0,0 +1,138 @@
+<?php
+
+/**
+ * @group query
+ * @group taxonomy
+ * @group meta
+ */
+class Test_Lazy_Load_Term_Meta extends WP_UnitTestCase {
+	/**
+	 * @var array
+	 */
+	protected static $post_ids = array();
+	/**
+	 * @var array
+	 */
+	protected static $term_ids = array();
+
+	public static function wpSetUpBeforeClass( WP_UnitTest_Factory $factory ) {
+		$post_type      = 'post';
+		self::$post_ids = $factory->post->create_many(
+			3,
+			array(
+				'post_type'   => $post_type,
+				'post_status' => 'publish',
+			)
+		);
+		$taxonomies     = get_object_taxonomies( $post_type, 'object' );
+		foreach ( self::$post_ids  as $post_id ) {
+			foreach ( $taxonomies as $taxonomy ) {
+				if ( ! $taxonomy->_builtin ) {
+					continue;
+				}
+				$terms          = $factory->term->create_many( 3, array( 'taxonomy' => $taxonomy->name ) );
+				self::$term_ids = array_merge( self::$term_ids, $terms );
+				foreach ( $terms as $term ) {
+					add_term_meta( $term, wp_rand(), 'test' );
+				}
+				wp_set_object_terms( $post_id, $terms, $taxonomy->name );
+			}
+		}
+	}
+
+	/**
+	 * @ticket 57150
+	 * @covers ::wp_queue_posts_for_term_meta_lazyload
```

Comment:
```suggestion
	 * @ticket 57150
	 *
	 * @covers ::wp_queue_posts_for_term_meta_lazyload
```

Per [56070](https://core.trac.wordpress.org/ticket/56070) should update this.